src/data/pokemon/species_info/gen_1.h

```diff
@@ -10184,11 +10178,18 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .trainerScale = 256,                                \
         .trainerOffset = 0,                                 \
         .footprint = gMonFootprint_Tauros,                  \
-        .formSpeciesIdTable = sTaurosFormSpeciesIdTable
+        .formSpeciesIdTable = sTaurosFormSpeciesIdTable     \
 
     [SPECIES_TAUROS] =
     {
         TAUROS_MISC_INFO,
+        .baseHP        = 75,                                \
+        .baseAttack    = 100,                               \
+        .baseDefense   = 95,                                \
+        .baseSpeed     = 110,                               \
+        .baseSpAttack  = 40,                                \
+        .baseSpDefense = 70,    
+        .baseSpDefense = 70,       
```

Comment:
Sorry, you need to remove the last comma here. :sweat_smile:

Reply:
damn i REALLY thought i had that

sorry for making you review something so simple 24 times😅

Reply:
> damn i REALLY thought i had that
> 
> sorry for making you review something so simple 24 times😅

It's fine but please compile before making a commit 😅

Reply:
sorry im not on my laptop im literally making the changes on github

Reply:
Please do it on a computer where you can compile. There is no rush :smile: